charts/seed-monitoring/charts/grafana/dashboards/operators/apiserver-watches-dashboard.json

```diff
@@ -0,0 +1,391 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 1,
+  "id": 22,
+  "iteration": 1597754908762,
+  "links": [],
+  "panels": [
+    {
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 4,
+      "panels": [],
+      "repeat": "Scope",
+      "scopedVars": {
+        "Scope": {
+          "selected": true,
+          "text": "cluster",
+          "value": "cluster"
+        }
+      },
+      "title": "Watch Durations for Scope: $Scope",
+      "type": "row"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "$Quantile of watches on this resource are less than or equal to the value.",
```

Comment:
I think it might be irritating to people, that in the normal/healthy case all watch duration metrics are at `60s`. Can you add a note here, that this is the maximum value recorded by the API server and `60s` means actually `[60s,+Inf)`?